src/codec/mod.rs

```diff
@@ -0,0 +1,188 @@
+use std::io::{Read, Write};
```

Comment:
This code is lifted verbatim from `src/net/codec.rs`, right?

Reply:
yes and `src/net/mod.rs`